test/cliargsparser.cpp

```diff
@@ -250,6 +250,60 @@ TEST_CASE(
 	}
 }
 
+TEST_CASE("Sets `cache_file` when provided combined short options",
```

Comment:
For this test, `cache_file` is just a detail — what it *really* tests is the support for combined short options. So I'd rename the test to "Supports combined short options".

Reply:
This will apply to the rust tests too right?

Reply:
Yep.